src/CoreOperators.ts

```diff
@@ -15,7 +15,7 @@ export interface CoreOperators<T> {
   combineAll?: <R>(project?: (...values: Array<any>) => R) => Observable<R>;
   combineLatest?: <R>(...observables: Array<Observable<any> | ((...values: Array<any>) => R)>) => Observable<R>;
   concat?: <R>(...observables: (Observable<any> | Scheduler)[]) => Observable<R>;
-  concatAll?: () => Observable<any>;
+  concatAll?: () => Observable<T>;
```

Comment:
I'm not a heavy or experimented TS user so I may be wrong but don't you lose the interest of TS if you put `<any>` here ?

My reasoning with these typings was more: let's inform the "compiler" that `switch` / `mergeAll` / `concatAll` have the interesting type behaviour to keep the type of the original observable sequence they derive from. This way the compiler can give better type informations. Wouldn't you lose all that with `<any>` ?